src/winbind.c

```diff
@@ -206,15 +206,76 @@ uint32_t winbind_cli_auth(char *user, char *domain,
     return ret;
 }
 
+/*  Worst-case SID in textual representation;
+    see https://stackoverflow.com/questions/1140528/
+        what-is-the-maximum-length-of-a-sid-in-sddl-format */
+static const size_t SID_MAX_STRING_LENGTH = 184 + 1;
+
+uint32_t put_sids_list_to_attributes(const struct wbcAuthUserInfo *wbc_info,
+                                     struct gssntlm_name_attribute **auth_attrs)
+{
+    static const char static_sids_urn[] = "urn:gssntlmssp:sids";
```

Comment:
We should define this one in src/gss_ntlmssp.c and src/gss_ntlmssp.h

Reply:
Ok